tsconfig-type-test.json

```
@@ -0,0 +1,22 @@
+{
+  "compilerOptions": {
+    "target": "es2022",
```

```suggestion
    "target": "es2021",
```

https://github.com/markedjs/marked/pull/2886